main.sh

```diff
@@ -328,8 +326,6 @@ then
     --cluster-ramsize="$DATA_QUOTA" \
     --cluster-index-ramsize="$INDEX_QUOTA" \
     --cluster-fts-ramsize="$SEARCH_QUOTA" \
-    --cluster-eventing-ramsize="$EVENTING_QUOTA" \
-    --cluster-analytics-ramsize="$ANALYTICS_QUOTA" \
```

Comment:
Removing the values that are not supported in CC @malscent correct me if I am wrong here